remote-build.exp

```diff
@@ -101,6 +100,12 @@ while { ${argi} < ${argc} } {
 				error "Could not parse ${arg}"
 			}
 		}
+		"--dir=[^ ]*" {
```

Comment:
It's a possibility too, I just didn't think there was a need for a 1:1 correspondence between internal variable names and parameter names. I'll change that in the next commit.